module/VuFindSearch/src/VuFindSearch/Backend/ProQuestFSG/Response/XML/RecordCollection.php

```diff
@@ -82,6 +89,19 @@ public function getTotal()
      */
     public function getFacets()
     {
-        return $this->response['facets'];
+        if ($this->facetFields === null) {
+            $this->facetFields = [];
+            $facets = $this->response['facets'] ?? [];
+            foreach ($facets as $facetName => $facetValues) {
+                usort($facetValues, fn ($a, $b) => ($a['count'] ?? 0) < ($b['count'] ?? 0));
+                $values = [];
+                foreach ($facetValues as $facetValue) {
+                    $facetValueName = "{$facetValue['code']}|{$facetValue['name']}";
```

Comment:
I used a `|` as the delimiter in the original ProQuestFSG code as well, and it works fine, but it does make the declaration of the delimiter in Blender.ini a little awkward:
```
[Advanced_Settings]
delimited_facets[] = "source||"
```
So could change it to something else if preferred.